tools/color-shapes/index.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import fs from 'fs/promises'
+import path from 'path'
+
+import { Space } from './detectColorSpace.js';
+import { makeColorSpaceGenerator } from './generate-color-space.js';
+import config from '../../config.js'
+import { makeColorShapeFileName } from './shape-name.js';
+
+// fix '"lch"' to 'lch'. Why so bad code...
+config.COLOR_FN = JSON.parse(config.COLOR_FN)
```

Comment:
```suggestion
const COLOR_TYPE = JSON.parse(config.COLOR_FN)
```

Reply:
no. As you can see, Im reading config directly here. 
I need config object with correct values for generation.
COLOR_FN: '"lch"' - is not correct 
COLOR_FN: 'lch' - is correct

Reply:
The problem is that you are changing the type of config. In this case you should rename the variable, because it it not that config enymore.